hooks/gen_docs/gen_docs_env_vars.py

```diff
@@ -21,7 +21,7 @@
 from hooks import PATH_ROOT
 from hooks.gen_docs import IterableStrEnum
 from kpops.cli import main
-from kpops.cli.pipeline_config import PipelineConfig
+from kpops.cli.config import KpopsConfig
```

Comment:
why is this in the `cli` module actually? should be just `kpops.config`, no?

Reply:
Moved it!